projects/core/src/lib/colors/color-palette-editor/color-palette-editor.component.scss

```diff
@@ -0,0 +1,49 @@
+.tabs-viewport {
+    display: flex;
+    height: 20rem;
+    flex-flow: row wrap;
+}
+
+geoengine-color-attribute-input {
+    flex-grow: 1;
+}
+
+.removeButton {
+    margin-left: 10px;
```

Comment:
Specify this in rem wrt. to other components that use 1rem or 0.5rem for such margins.